### PR TITLE
fix: Remove Cannot resolve symbol from the integration tests in v2

### PR DESCRIPTION
**Pull request by JerryWu1234**
# Overview

https://github.com/orgs/QwikDev/projects/1/views/1?pane=issue&itemId=68407338

# What is it?

- [x] Feature / enhancement
- [x] Bug
- [x] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/QwikDev/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] Added new tests to cover the fix / functionality

### CODE REVIEW

**Review by Varixo — CHANGES_REQUESTED** [review]
This is not the correct fix. You shoudn't remove the error message from the code. Instead you need to find why this message is appearing and do something inside the tests

**Comment by JerryWu1234** [comment]
image

>This is not the correct fix. You shoudn't remove the error message from the code. Instead you need to find why this message is appearing and do something inside the tests

Sorry, I got it wrong.

I finish it now. I tweak the unit tests

File: packages/qwik/src/core/v2/tests/render-api.spec.tsx [hunk 1]
```diff
@@ -46,15 +46,16 @@ const defaultManifest: QwikManifest = {
...
+    s_click: 's_click.js',
+  },
   version: '1',
 };
 
 const defaultCounterManifest: QwikManifest = {
   ...defaultManifest,
-  mapping: {
-    click: 'click.js',
-  },
```
Comment (wmertens) [inline packages/qwik/src/core/v2/tests/render-api.spec.tsx:57]: If you're moving everything into the default manifest, the countermanifest is no longer useful, so you should remove it.

I think it's fine to do it, the manifest may contain entries that aren't used.

**Comment by wmertens** [comment]
Please fix the formatting. I recommend you enable format-on-save in your editor, with Prettier support enabled.

**Review by wmertens — APPROVED** [review]
LGTM

**Pull request merged**